scripts/o3de/o3de/export_project.py

```diff
@@ -403,6 +406,8 @@ def build_export_toolchain(ctx: O3DEScriptExportContext,
         cmake_configure_command.extend([f'-DCMAKE_BUILD_TYPE={PREREQUISITE_TOOL_BUILD_CONFIG}'])
     if engine_centric:
         cmake_configure_command.extend([f'-DLY_PROJECTS={ctx.project_path}'])
+    if additional_cmake_configure_options:
+        cmake_configure_command.extend([f'-D{option}' for option in additional_cmake_configure_options])
```

Comment:
Why not pass the option as is?

That would make it more flexible if for some reason, I have a cmake preset that defines a bunch of Cache Variables/Environment variables
```suggestion
    if additional_cmake_configure_options:
        cmake_configure_command.extend(additional_cmake_configure_options)
```

Reply:
Can we perform similar logic to what we performed with the `/` slash option for the ` -cco` option?

Reply:
I can't think of a clean way to represent that. Maybe have the first '/' represent the config options and any trailing one after that represents the --build options?